sales.html

```diff
@@ -35,17 +35,17 @@
         Max Customers: <input type="number" name="Max Customers" />
       </p>
       <p>
-        Average Cookie Sale: <input type="number" name="Average Cookie Sale" />
+        Average Cookie Sale: <input type="number" name="Average Cookie Sale" step="0.1" />
       </p>
       <p>
-        Hour Open: <input type="number" name="Hour Open" />
+        Hour Open: <input type="number" name="Hour Open" min="1" max="12" value="6" step="0.5"/>
```

Comment:
Half hours, eh? Tricky... do you plan to handle that?

I love the intent behind `max="12"` + AM/PM `select`, but I'd start with 0–23 and reintroduce AM/PM as a stretch goal.